qiita_pet/support_files/doc/source/downloading.rst

```diff
@@ -47,6 +58,42 @@ appear below. These files can include distance matrixes, PCoA, visualization,
 other resulting files. These can be download from here and further analyzed
 outside of Qiita.
 
+Access a public study or artifact without login
+-----------------------------------------------
+
+**no log required / wget or curl friendly**
+
+To display the study or artifact information within Qiita you can (do not
+forget to replace study-id or artifact-id):
+
+- Study: https://qiita.ucsd.edu/public/?study_id=study-id
+- Artifact: https://qiita.ucsd.edu/public/?artifact_id=artifact-id
+
+Now, if you would like to download any give artifact you can (do not forget to
+replace artifact-id):
+
+- https://qiita.ucsd.edu/public_artifact_download/?artifact_id=artifact-id
+
+Download raw or all BIOM files from a study
+-------------------------------------------
+
+**no log required / wget or curl friendly**
+
+We provide direct access to public data via a single end point. This end point
+can be used to download BIOMs or raw data, in specific (do not forget to
```

Comment:
"... or raw data. Do not forget ..." And make sure to remove the closing parentheses